template/App.js

```diff
@@ -6,44 +6,83 @@
  * @flow
  */
 
-import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
+import React, {Fragment} from 'react';
+import {StyleSheet, ScrollView, View, Text, StatusBar} from 'react-native';
 
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android:
-    'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
+import {
+  Header,
+  LearnMoreLinks,
+  Colors,
+  DebugInstructions,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
```

Comment:
this definitely needs an annotation that importing stuff from `react-native/Libraries` is only for demonstration purposes and one should never do it in a real app because it may break.